context/app/static/js/components/home/ExploreTools/ExploreTools.tsx

```diff
@@ -1,13 +1,104 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-import { Typography } from '@mui/material';
+import Grid from '@mui/material/Unstable_Grid2';
+import { useTheme } from '@mui/material/styles';
+import { entityIconMap } from 'js/shared-styles/icons/entityIconMap';
+import { ToolsCard } from './ToolsCard';
+import { CardGridContextProvider } from './CardGridContext';
+import { ToolDescription } from './ToolDescription';
+import { makeGridTemplateColumns } from './utils';
+
+const workspaceCTAAuthenticated = {
+  ctaText: 'View Your Workspaces',
+  ctaLink: '/workspaces',
+  ctaIcon: <entityIconMap.Workspace />,
+};
+
+const workspaceCTAGuest = {
+  ctaText: 'Sign in',
+  ctaLink: '/login',
+};
+
+const cards = [
+  {
+    title: 'Analyze data in Workspaces',
+    src: `${CDN_URL}/v2/explore-tools/workspaces.png`,
+    expandedContent: (
+      <ToolDescription
+        subtitle="Load datasets into an interactive JupyterLab Python and R analysis environment."
+        checklistItems={[
+          'No need to download data.',
+          'Use the provided code templates to get started with HuBMAP data.',
+          'The Workspaces feature is available once you sign in.',
+        ]}
+        {...(isAuthenticated ? workspaceCTAAuthenticated : workspaceCTAGuest)}
+      />
+    ),
+  },
+  {
+    title: 'Visualize data in Vitessce',
+    src: `${CDN_URL}/v2/explore-tools/vitessce.png`,
+    expandedContent: (
```

Comment:
The `ToolDescription` is in each of these, should it just be moved into the `ToolsCard`?

Reply:
I put `ToolDescription` in `ExploreTools` instead in order to make `ToolsCard` easy to reuse if we end up adding this card grid pattern elsewhere.